packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -611,41 +595,39 @@ describe('account', () => {
                             data {
-                                parsed {
-                                    info {
-                                        authorizedVoters {
-                                            authorizedVoter {
-                                                address
-                                            }
-                                            epoch
-                                        }
-                                        authorizedWithdrawer {
-                                            address
-                                        }
-                                        commission
-                                        epochCredits {
-                                            credits
-                                            epoch
-                                            previousCredits
-                                        }
-                                        lastTimestamp {
-                                            slot
-                                            timestamp
-                                        }
-                                        node {
-                                            address
-                                        }
-                                        priorVoters
-                                        rootSlot
-                                        votes {
-                                            confirmationCount
-                                            slot
-                                        }
+                                authorizedVoters {
+                                    authorizedVoter {
+                                        address
                                     }
-                                    type
+                                    epoch
+                                }
+                                authorizedWithdrawer {
+                                    address
+                                }
+                                commission
+                                epochCredits {
+                                    credits
+                                    epoch
+                                    previousCredits
                                 }
+                                lastTimestamp {
+                                    slot
+                                    timestamp
+                                }
+                                node {
+                                    address
+                                }
+                                priorVoters
+                                rootSlot
+                                votes {
+                                    confirmationCount
+                                    slot
+                                }
+                            }
+                            meta {
                                 program
```

Comment:
What should we call this for real in this greenfield schema?

* Assigned Program id
* Owner
* Owning program

https://docs.solana.com/developing/programming-model/accounts#ownership-and-assignment-to-programs

Reply:
`ownerProgram` I think.